pkg/acsengine/pki.go

```diff
@@ -68,6 +72,18 @@ func CreatePki(extraFQDNs []string, extraIPs []net.IP, clusterDomain string, caP
 
 	go func() {
 		var err error
```

Comment:
so we should only need the ip addresses. As we are calling into the etcd cluster by ip addresses we shouldn't need the fqdns. Can you set that to nil.